src/common/localization/zh-ch.ts

```diff
@@ -0,0 +1,509 @@
+        },
+        videoSettings: {
+            title: "视频设定", // Video Settings
+            description: "提取帧以进行标记的速率", // The rate at which frames are extracted for tagging.
+            frameExtractionRate: "帧提取率（每视频每秒的帧数）", // Frame Extraction Rate (frames per a video second)
+        },
+        addConnection: "添加连接", // Add Connection
+        messages: {
+            saveSuccess: "成功保存$ {project.name}项目设置", // Successfully saved ${project.name} project settings
+        },
+    },
+    projectMetrics: {
+        title: "项目指标", // Project Metrics
+        assetsSectionTitle: "资产", // Assets
+        totalAssetCount: "总资产", // Total Assets
+        visitedAssets: "访问的资产（$ {count}）", // Visited Assets (${count})
+        taggedAssets: "标记资产（$ {count}）", // Tagged Assets (${count})
+        nonTaggedAssets: "未标记的资产（$ {count}）", // Not Tagged Assets (${count})
+        nonVisitedAssets: "未访问的资产（$ {count}）", // Not Visited Assets (${count})
+        tagsSectionTitle: "标签和标签", // Tags & Labels
+        totalRegionCount: "标记区域总数", // Total Tagged Regions
+        totalTagCount: "总标签", // Total Tags
+        avgTagCountPerAsset: "每个资产的平均标签", // Average tags per asset
+    },
+    tags: {
+        title: "标签", // Tags
+        placeholder: "新增标签", // Add new tag
+        editor: "标签编辑器", // Tags Editor
+        modal: {
+            name: "标签名称", // Tag Name
+            color: "标签颜色", // Tag Color
+        },
+        colors: {
+            white: "白色", // White
+            gray: "灰色", // Gray
+            red: "红色", // Red
+            maroon: "栗色", // Maroon
+            yellow: "黄色", // Yellow
+            olive: "橄榄色", // Olive
+            lime: "青色", // Lime
+            green: "绿色", // Green
+            aqua: "水色", // Aqua
+            teal: "蓝绿色", // Teal
+            blue: "蓝色", // Blue
+            navy: "海军蓝色", // Navy
+            fuschia: "紫红色", // Fuschia
+            purple: "紫色", // Purple
+        },
+        warnings: {
+            existingName: "标签名称已存在。请选择另一个名字", // Tag name already exists. Choose another name
+            emptyName: "标签名称不能为空", // Cannot have an empty tag name
+            unknownTagName: "未知", // Unknown
+        },
+        toolbar: {
+            add: "新增标签", // Add new tag
+            search: "搜索标签", // Search tags
+            edit: "编辑标签", // Edit tag
+            lock: "锁定标签", // Lock tag
+            moveUp: "向上移动标签", // Move tag up
+            moveDown: "向下移动标签", // Move tag down
+            delete: "删除标签", // Delete tag
+        },
+    },
+    connections: {
+        title: "连接数", // Connections
+        details: "连接细节", // Connection Details
+        settings: "连接设定", // Connection Settings
+        instructions: "请选择一个连接进行编辑", // Please select a connection to edit
+        save: "保存连接", // Save Connection
+        messages: {
+            saveSuccess: "已成功保存$ {connection.name}", // Successfully saved ${connection.name}
```

Comment:
there should not be a space between `$` and `${connection.name}`.  Same for everywhere else a variable is use.

Reply:
@mydiemho Thank you for reviewing very carefully. I just updated it to fix variables.